03/MenuItem.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+const MenuItem = (props) => {
+    return(
```

Comment:
Warto też wykorzystywać destrukturyzację tj. `const {url, text} = props;` i potem te zmienne przekazywać - przy większych komponentach jest to czytelniejsze.